raco/expression/function.py

```diff
@@ -66,8 +116,19 @@ def typeof(self, scheme, state_scheme):
         return types.DOUBLE_TYPE
 
 
+class YEAR(UnaryFunction):
```

Comment:
It was a quick and dirty way to support YEAR, which was needed [in TPC-H queries](https://github.com/uwescience/tpch-radish/blob/7d8e8a8ef5808a6ceccc5d9c4e61b6df7d40f60d/templates/9.sql). I'll add at least the others that make sense.